Sources/NIOWebSocket/WebSocketFrameDecoder.swift

```diff
@@ -270,6 +270,7 @@ public final class WebSocketFrameDecoder: ByteToMessageDecoder {
     }
 
     public func decodeLast(context: ChannelHandlerContext, buffer: inout ByteBuffer, seenEOF: Bool) throws -> DecodingState {
```

Comment:
I think, we can remove this method definition which is same as the default implementation. Am I right ?

Reply:
Yes, agreed